src/cljs/rems/user_settings.cljs

```diff
@@ -7,8 +7,16 @@
 (rf/reg-event-fx
  :loaded-user-settings
  (fn [{:keys [db]} [_ user-settings]]
-   (language/update-language (:language user-settings))
-   {:db (assoc db :user-settings user-settings)}))
+   (let [new-language (or (language/get-language-cookie)
+                          (:language user-settings))
```

Comment:
hmm so we have the overriding here and in the `:language` subscription... could we get rid of the other?